cli/lsp/resolver.rs

```diff
@@ -181,8 +196,18 @@ impl LspResolver {
     Ok(())
   }
 
-  pub fn as_graph_resolver(&self) -> &dyn Resolver {
-    self.graph_resolver.as_ref()
+  pub fn as_graph_resolver<'a, 'b>(
+    &'a self,
+    // this really only needs a HashSet<ModuleSpecifier>, but it's provided
+    // the entire HashMap to avoid cloning all the time
+    open_docs: &'b HashMap<ModuleSpecifier, Arc<Document>>,
```

Comment:
Was thinking about this more this morning and realized that we should just not bother passing the open documents here. In places like vscode, the file will always exist on the file system if it's open (unlike our test server), so we can just consult that for simplicity.